docs/docs.go

```diff
@@ -0,0 +1,360 @@
+                    "type": "string"
+                },
+                "version": {
+                    "description": "Version identifies the revision number for this struct.",
+                    "type": "string"
+                }
+            }
+        },
+        "types.ConditionKind": {
+            "type": "string",
+            "enum": [
+                "firmwareInstall",
+                "inventoryOutofband"
+            ],
+            "x-enum-varnames": [
+                "FirmwareInstall",
+                "InventoryOutofband"
+            ]
+        },
+        "types.ConditionState": {
+            "type": "string",
+            "enum": [
+                "pending",
+                "active",
+                "failed",
+                "succeeded"
+            ],
+            "x-enum-varnames": [
+                "Pending",
+                "Active",
+                "Failed",
+                "Succeeded"
+            ]
+        },
+        "types.ConditionsResponse": {
+            "type": "object",
+            "properties": {
+                "conditions": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/types.Condition"
+                    }
+                },
+                "serverID": {
+                    "type": "string"
+                }
+            }
+        },
+        "types.Fault": {
+            "type": "object",
+            "properties": {
+                "delayDuration": {
+                    "description": "Introduce specified delay in execution of the condition on the controller.\n\naccepts the string format of time.Duration - 5s, 5m, 5h",
+                    "type": "string"
+                },
+                "failAt": {
+                    "description": "FailAt is a controller specific task/stage that the condition should fail in execution.\n\nfor example, in the flasher controller, setting this field to ` + "`" + `init` + "`" + ` will cause the\ncondition task to fail at initialization.",
+                    "type": "string"
+                },
+                "panic": {
+                    "description": "will cause the condition execution to panic on the controller.",
+                    "type": "boolean"
+                }
+            }
+        },
+        "types.ServerResponse": {
+            "type": "object",
+            "properties": {
+                "message": {
+                    "type": "string"
+                },
+                "records": {
+                    "$ref": "#/definitions/types.ConditionsResponse"
+                },
+                "statusCode": {
+                    "type": "integer"
+                }
+            }
+        }
+    }
+}`
+
+// SwaggerInfo holds exported Swagger Info so clients can modify it
+var SwaggerInfo = &swag.Spec{
+	Version:          "1.0.0",
+	Host:             "",
+	BasePath:         "/api",
```

Comment:
shouldn't `BasePath` be set to `/api/v1`

Reply:
If that’s something we control (I just noticed it’s a `&swag.Spec`), then we likely have control over both version and the path the api is hosted at, which might be best as `/api/v1/docs`?

Reply:
yes